pkg/cmd/stack/deploy.go

```diff
@@ -85,8 +85,30 @@ const (
 	maxRestartsToConsiderFailed = 3
 )
 
+func (sd *Stack) RunDeploy(ctx context.Context, s *model.Stack, options *DeployOptions) error {
```

Comment:
if stack is gone, can we rename this to Compose or similar?

Reply:
We removed the command but not the syntax so I think it's ok to leave the name like this